Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -59,9 +59,10 @@
                           standalonemuontracking*
                           trackingGlobalReco*
                           vertexreco)
-_globalreco_tracking_Phase1PU70 = globalreco_tracking.copy()
-_globalreco_tracking_Phase1PU70.replace(trackingGlobalReco, recopixelvertexing+trackingGlobalReco)
-eras.trackingPhase1PU70.toReplaceWith(globalreco_tracking, _globalreco_tracking_Phase1PU70)
+_globalreco_tracking_LowPU_Phase1PU70 = globalreco_tracking.copy()
```

Comment:
@slava77 OK. Would it be fine to leave the update to the point where trackingPhase1PU70 is cleaned up, or would you like to see it addressed sooner?